yb-voyager/cmd/importSchema.go

```diff
@@ -112,12 +112,16 @@ func importSchema() error {
 			utils.ErrExit("Failed to get missing import schema permissions: %s", err)
 		}
 		if len(missingPermissions) > 0 {
-			utils.PrintAndLog(color.RedString("The target database is missing the following permissions required for importing schema:"))
+			// Not printing the target db is missing permissions message for YB
+			// In YB (the only db allowed in import schema) we only check whether the user is a superuser and hence we don't need to print the message
+			if source.DBType == YUGABYTEDB {
```

Comment:
Since we only import schema to YB, i don't understand why we need this if check. Just do whatever we'd want in the case of YB?